src/patterns/dosdp-patterns/boneLengthAlongAxis.yaml

```diff
@@ -0,0 +1,103 @@
+---
+pattern_name: boneLengthAlongAxis
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/boneLengthAlongAxis.yaml
```

Comment:
Lets use snake_case for all pattern names, like the rest: https://github.com/obophenotype/bio-attribute-ontology/tree/master/src/patterns/dosdp-patterns

Also rename file accordingly.